### PR TITLE
Fix extensions in test headers

### LINKED ISSUE #340: Test comment headers outdated?
This repo has already been really useful for developing an action from scratch - I see it's being actively developed again which is great.

The header comments on the test files seem to be outdated now - they mention `.ts` files, and the main one mentions setting inputs as environment variables:

```
/**
 * Unit tests for the action's main functionality, src/main.ts
 *
 * These should be run as if the action was called from a workflow.
 * Specifically, the inputs listed in `action.yml` should be set as environment
 * variables following the pattern `INPUT_<INPUT_NAME>`.
 */
 ```
 
 However as far as I can tell this isn't necessary, is that right?

### PR DESCRIPTION
**Pull request by ncalteen**
This PR updates the file extensions in the tests to reference the correct files (JavaScript, not TypeScript).

Fixes #340

### CODE REVIEW

File: __tests__/wait.test.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Unit tests for src/wait.ts
+ * Unit tests for src/wait.js
```
Comment (sethrylan) [inline __tests__/wait.test.js:2]: suggestion: I think we should also update line 7 (just outside the PR diff)

```
describe('wait.ts', () => {
```

Comment (ncalteen) [reply __tests__/wait.test.js:2]: Thanks for catching that one!

Comment (ncalteen) [reply __tests__/wait.test.js:2]: Updated!

**Review by sethrylan — APPROVED** [review]
LGTM

**Pull request merged**